content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -82,18 +82,42 @@ packages that define the API objects.
 
 ### OpenAPI V3
 
-{{< feature-state state="alpha"  for_k8s_version="v1.23" >}}
+{{< feature-state state="beta"  for_k8s_version="v1.24" >}}
 
-Kubernetes v1.23 offers initial support for publishing its APIs as OpenAPI v3; this is an
-alpha feature that is disabled by default.
-You can enable the alpha feature by turning on the
+Kubernetes v1.24 offers beta support for publishing its APIs as OpenAPI v3; this is a
+beta feature that is enabled by default.
+You can disable the beta feature by turning off the
 [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) named `OpenAPIV3`
 for the kube-apiserver component.
 
-With the feature enabled, the Kubernetes API server serves an
-aggregated OpenAPI v3 spec per Kubernetes group version at the
-`/openapi/v3/apis/<group>/<version>` endpoint. Please refer to the
-table below for accepted request headers.
+A discovery endpoint `/openapi/v3` is provided to see a list of all
+group/versions available. This endpoint only returns JSON. These group/versions
+are provided in the following format:
+```
+{
+  "paths": [{
+    "$group_version": {
+      "serverRelativeURL": "$relative_url?hash=$current_resource_hash"
+	},
+	"apps/v1": {
+	  "serverRelativeURL": "/openapi/v3/apis/apps/v1?hash=ha5hha5hha5hha5h"
```

Comment:
Is this taken from a real response? I feel like it should be `v1.json` here. If it is written by hand, maybe there are other mistakes I am not catching as well.

Reply:
Thanks, I fixed all these comments by grabbing an example from the server